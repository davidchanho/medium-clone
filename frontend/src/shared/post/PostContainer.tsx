import React, { FC } from "react";
import { Card } from "react-bootstrap";
import { PostContainerProps } from "./types";


const PostContainer: FC<PostContainerProps> = ({ children, onClick }) => {
  return (
    <Card
      className="d-flex flex-column bg-inherit pointer mb-2"
      onClick={onClick}
    >
      {children}
    </Card>
  );
};

export default PostContainer;
