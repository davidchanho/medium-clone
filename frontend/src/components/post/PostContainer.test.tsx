import React from "react";
import renderer from "react-test-renderer";
import PostContainer from "./PostContainer";

const onClick = () => {};

describe("PostContainer", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<PostContainer onClick={onClick}> test</PostContainer>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
