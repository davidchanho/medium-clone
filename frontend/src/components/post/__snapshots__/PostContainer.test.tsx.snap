// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostContainer renders correctly 1`] = `
<div
  className="d-flex flex-column bg-inherit pointer mb-2 card"
  onClick={[Function]}
>
   test
</div>
`;
